cmd/admin-ui/src/routes/components/Header.svelte

```diff
@@ -0,0 +1,10 @@
+<div class="w-full h-16 bg-slate-800 fixed top-0">
+	<div class="w-[90%] h-full flex mx-auto items-center justify-between text-white">
+		<h1 class="rounded-[3px] text-[20px] bg-[#2f415e] p-1">
```

Comment:
`#2f415e` could be added as a custom colour in the tailwind config, but probably not needed at the moment